IntegrationTests/BTCardClient_IntegrationTests.swift

```diff
@@ -31,8 +33,10 @@ class BTCardClient_IntegrationTests: XCTestCase {
     }
 
     func testTokenizeCard_whenCardIsInvalidAndValidationIsEnabled_failsWithExpectedValidationError() {
-        let apiClient = BTAPIClient(authorization: BTIntegrationTestsConstants.sandboxClientToken)!
-        let cardClient = BTCardClient(apiClient: apiClient)
+        var apiClient = BTAPIClient(authorization: BTIntegrationTestsConstants.sandboxClientToken)!
+        var cardClient = BTCardClient(authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn")
```

Comment:
Can we update all the card clients in this file to use the same auth string as the apiClient?

Reply:
yep, updated in 9e673a83fbde610930207c112a4aa353fbbc09be